lib/hydra/derivatives/services/persist_external_file_output_file_service.rb

```diff
@@ -0,0 +1,18 @@
+module Hydra::Derivatives
+  class PersistExternalFileOutputFileService < PersistOutputFileService
+    # Persists a new file object that points to external content
+    # @param [Hash] output information about the external derivative file
+    # @option output [String] url the location of the external content
+    # @param [Hash] directives directions which can be used to determine where to persist to.
+    # @option directives [String] url This can determine the path of the object.
```

Comment:
The "object" that this comment refers to is the parent object that the new `ActiveFedora::File` record will be attached to, right?  Could you change the comment to make it more clear what that "object" refers to?